tutorpicasso/commands/run_extra_commands.py

```diff
@@ -80,6 +81,7 @@ def run_command(command: str) -> None:
     Args:
         command (str): Tutor command.
     """
+    click.echo(tutor_fmt.command(command))
```

Comment:
I don't want this to be a blocker since the design of running multiple commands is out of the scope of this PR. However, I wonder why using `with subprocess.Popen` instead of running all commands with `tutor_utils.execute` to save ourselves from manually error handling the process output.

Reply:
I tried, but it makes the chained elements weird.

For example:
Using: `tutor_utils.execute(*command.split())` it returns:
`tutor plugins update '|' tutor plugins index add https://raw.githubusercontent.com/eduNEXT/tutor-plugin-indexes/picasso_test/`
To make it possible to allow chained elements, we would need to add shell=True in https://github.com/overhangio/tutor/blob/master/tutor/utils.py#L216

Reply:
I found a way:
Using: `tutor_utils.execute('sh', '-c', command)`
The only thing that is not so good about this is that every command will have a sh -c prefix:
![image](https://github.com/user-attachments/assets/7195b0bc-48cc-4d3d-82ac-a7084fbcc8bc)

Reply:
I used this solution. Having the sh -c prefix is not good looking but improves the readability of the code and doesn't bother a lot in the logs.

If you have more feedback about this, please let me know.

Reply:
This looks significantly better and gives even more information for debugging purposes. Thank you!